rest/taskana-rest-spring/src/main/java/pro/taskana/common/rest/TaskanaRestExceptionHandler.java

```diff
@@ -33,113 +35,113 @@
-      DuplicateKeyException ex, WebRequest req) {
+  protected ResponseEntity<Object> handleDuplicateKey(DuplicateKeyException ex, WebRequest req) {
     return buildResponse(ex, req, HttpStatus.CONFLICT);
   }
 
   @ExceptionHandler(ConcurrencyException.class)
-  protected ResponseEntity<TaskanaErrorData> handleConcurrencyException(
+  protected ResponseEntity<Object> handleConcurrencyException(
       ConcurrencyException ex, WebRequest req) {
     return buildResponse(ex, req, HttpStatus.CONFLICT);
   }
 
   @ExceptionHandler(WorkbasketInUseException.class)
-  protected ResponseEntity<TaskanaErrorData> handleWorkbasketInUse(
+  protected ResponseEntity<Object> handleWorkbasketInUse(
       WorkbasketInUseException ex, WebRequest req) {
     return buildResponse(ex, req, HttpStatus.LOCKED);
   }
 
   @ExceptionHandler(WorkbasketAlreadyExistException.class)
-  protected ResponseEntity<TaskanaErrorData> handleWorkbasketAlreadyExist(
+  protected ResponseEntity<Object> handleWorkbasketAlreadyExist(
       WorkbasketAlreadyExistException ex, WebRequest req) {
     return buildResponse(ex, req, HttpStatus.CONFLICT);
   }
 
   @ExceptionHandler(WorkbasketAccessItemAlreadyExistException.class)
-  protected ResponseEntity<TaskanaErrorData> handleWorkbasketAccessItemAlreadyExist(
+  protected ResponseEntity<Object> handleWorkbasketAccessItemAlreadyExist(
       WorkbasketAccessItemAlreadyExistException ex, WebRequest req) {
     return buildResponse(ex, req, HttpStatus.CONFLICT);
   }
 
   @ExceptionHandler(InvalidWorkbasketException.class)
-  protected ResponseEntity<TaskanaErrorData> handleInvalidWorkbasket(
+  protected ResponseEntity<Object> handleInvalidWorkbasket(
       InvalidWorkbasketException ex, WebRequest req) {
     return buildResponse(ex, req, HttpStatus.BAD_REQUEST);
   }
 
   @ExceptionHandler(DomainNotFoundException.class)
-  protected ResponseEntity<TaskanaErrorData> handleDomainNotFound(
+  protected ResponseEntity<Object> handleDomainNotFound(
       DomainNotFoundException ex, WebRequest req) {
     return buildResponse(ex, req, HttpStatus.BAD_REQUEST);
   }
 
   @ExceptionHandler(MaxUploadSizeExceededException.class)
-  protected ResponseEntity<TaskanaErrorData> handleMaxUploadSizeExceededException(
+  protected ResponseEntity<Object> handleMaxUploadSizeExceededException(
       MaxUploadSizeExceededException ex, WebRequest req) {
     return buildResponse(ex, req, HttpStatus.PAYLOAD_TOO_LARGE);
   }
 
+  @Override
+  protected ResponseEntity<Object> handleBindException(
+      BindException ex, HttpHeaders headers, HttpStatus status, WebRequest request) {
+    return buildResponse(ex, request, HttpStatus.BAD_REQUEST);
+  }
+
   @ExceptionHandler(Exception.class)
-  protected ResponseEntity<TaskanaErrorData> handleGeneralException(Exception ex, WebRequest req) {
+  protected ResponseEntity<Object> handleGeneralException(Exception ex, WebRequest req) {
     return buildResponse(ex, req, HttpStatus.BAD_REQUEST);
   }
```

Comment:
Did you test that this is working?

Reply:
Yes, it works as we predicted for Taskana Exceptions which are not mentioned explicitly before.